src/spikeinterface/comparison/basecomparison.py

```diff
@@ -96,6 +99,10 @@ def _do_comparison(
             print("Multicomparison step 1: pairwise comparison")
 
         self.comparisons = {}
+
+        n_jobs = fix_job_kwargs({"n_jobs": self.n_jobs})["n_jobs"]
```

Comment:
Here this will not have the general behavior.
In short if a user do
```python
set_global_job_kwargs(n_jobs=-1)
comp  = BaseMultiComparison()
```
then the n_jobs will be 1 and so not parralel.